src/main.py

```
@@ -1,6 +1,8 @@
 import sys
 import polars as pl
 
+from demoparser2 import DemoParser
```

Flake8 (7.2.0)

[error] 'demoparser2.DemoParser' imported but unused

(F401)